docs/modules/ROOT/pages/development.adoc

```diff
@@ -426,6 +426,22 @@ end
 
 You can specify any gem requirement using https://guides.rubygems.org/patterns/#declaring-dependencies[the same syntax as your `Gemfile`].
 
+You can also handle multiple versions of a gem with `target_gem_version`. It behaves similar to `target_ruby_version`, allowing you to inspect a gem version at runtime:
+
+```Ruby
+class MyCop < Base
+  requires_gem "my-gem"
```

Comment:
This way `target_gem_version` will not return nil (the cop would not run in that case).